CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v2.0.1
```

Comment:
I think we should probably not include this in the release notes, as the documentation doesn't get bundled up and released to PyPI (it's only the stuff in the `varinfo` directory that gets included in a release).

Reply:
Thanks! Haha I didn't know this lol